routes/api/v1/playlists.js

```diff
@@ -40,6 +40,36 @@ router.post('/', (request, response) => {
     }
 });
 
+router.put('/:id', (request, response) => {
```

Comment:
Functionality looks great. Hopefully, we can refactor to eliminate multiple database calls next.